app/models/app.rb

```diff
@@ -2,6 +2,7 @@
 
 class App < ApplicationRecord
   belongs_to :team
+  has_many :deployments, dependent: :nullify
```

Comment:
Should this dependent be `:destroy`? Otherwise when an app is destroyed we will have a bunch of deployments hanging around with no app associated.

Reply:
Yes! Just pushed up this change too